docker-compose.yml

```diff
@@ -2,7 +2,6 @@ version: '3.8'
 
 services:
   web:
-    image: datacite/viringo
```

Comment:
Is there a reason for removing this vs forcing the build when required via docker compose arguments?

Reply:
I didn't want to accidentally pull down the dockerhub image instead of building....not that I did that...multiple times